less/ng-obiba-mica.less

```diff
@@ -389,7 +389,12 @@ span.search-help {
 }
 
 .obiba_mica_lists {
+  .caption-count{
+    position: absolute;
+    bottom: 0;
+  }
   .read-more-btn{
```

Comment:
In bootstrap there is the "label" style. Use it instead of reinventing the wheel.

Reply:
@ymarcon 
This rule is to add some margin button to the read more link, 
the label style add more complexity to fix some vertical alignment issues

Reply:
Then you should apply extra style on the label and not creating a label-like rule.